engine/swarm/services.md

```diff
@@ -893,7 +893,7 @@ $ docker service create --name=my_redis \
 ### Give a service access to volumes or bind mounts
 
 For best performance and portability, you should avoid writing important data
-directly into a container's writable layer, instead using data volumes or bind
+directly into a container's writable layer, using instead data volumes or bind
```

Comment:
```suggestion
directly into a container's writable layer, and instead use data volumes or bind
```

Reply:
I like "using instead" better :)
Alternatively, we could split this into two sentences. "...you should avoid writing...layer. Instead, you should use..."
I leave this one to you to decide.

Reply:
Splitting into two sentences SGTM!